pulsar-io/kafka-connect-adaptor/src/main/java/org/apache/pulsar/io/kafka/connect/PulsarKafkaConnectSinkConfig.java

```diff
@@ -73,6 +73,14 @@
             help = "In case of Record<KeyValue<>> data use key from KeyValue<> instead of one from Record.")
     private boolean unwrapKeyValueIfAvailable = true;
 
+    @FieldDoc(
+            defaultValue = "false",
+            help = "Some connectors cannot handle pulsar topic names like persistent://a/b/topic"
+                    + " and do not sanitize the topic name themselves. \n"
+                    + "If enabled, all non alpha-digital characters in topic name will be replaced with underscores. \n"
+                    + "In some cases it may result in topic name collisions (topic_a and topic.a will become the same)")
+    private boolean sanitizeTopicName = false;
```

Comment:
It might be worth adding a note in the `topic` field to describe how `sanitizeTopicName` can be used to address valid Pulsar topic names that are invalid in Kafka.

Reply:
1. I don't think it works now (there was someone's unmerged PR to fix that IIRC)

2. it does not work for the sink that subscribes to multiple topics if you want to route topics into their respective destination and not mix all together

```
$ bin/pulsar-admin sinks
...
          -i, --inputs
            The sink's input topic or topics (multiple topics can be specified
            as a comma-separated list)
...
          --topics-pattern
            TopicsPattern to consume from list of topics under a namespace
            that match the pattern. [--input] and [--topicsPattern] are
            mutually exclusive. Add SerDe class name for a pattern in
```

Reply:
Thanks for clarifying.